lib/liquid/combinators/tags/if.ex

```diff
@@ -28,8 +28,40 @@ defmodule Liquid.Combinators.Tags.If do
     |> optional(parsec(:__parse__))
   end
 
+  @type unless_tag :: [If.t()]
```

Comment:
`unless` is not `If.t()` because it doesn't returns same AST like `if`, it only returns `[unless: Liquid.t()]`